conf/defaults.ini

```diff
@@ -521,6 +521,16 @@ nodata_or_nullvalues = no_data
 # This limit will protect the server from render overloading and make sure notifications are sent out quickly
 concurrent_render_limit = 5
 
+## Default setting for alert calculation timeout
```

Comment:
Should only be 1 #-character, not two. Two other places below with this problem.

Reply:
My fault, you are really very attentive